packages/react-shortcuts/README.md

```diff
@@ -0,0 +1,156 @@
+# `@shopify/react-shortcuts`
+
+![bundle size badge](https://img.shields.io/bundlephobia/minzip/@shopify/react-shortcuts.svg)
+
+Declaratively and efficiently match shortcut combinations in your react application.
+
+## Installation
+
+```bash
+$ yarn add @shopify/react-shortcuts
+```
+
+## Usage
+
+The library exposes three main parts, `<ShortcutProvider />`, `<Shortcut />` and `ShortcutManager`.
```

Comment:
Since `ShortcutManager` is only used internally and not by the consumer, should we just not mention it?

Reply:
(see web's app-context mock)